src/jit/codegencommon.cpp

```diff
@@ -10383,10 +10383,6 @@ void CodeGen::genSetScopeInfo()
 
         Compiler::siVarLoc varLoc;
 
-        // TODO-Review: This only works for always-enregistered variables. With LSRA, a variable might be in a register
-        // for part of its lifetime, or in different registers for different parts of its lifetime.
-        // This should only matter for non-debug code, where we do variable enregistration.
-        // We should store the ranges of variable enregistration in the scope table.
```

Comment:
I'm not sure this comment is obsolete. As far as I can tell, we still only report variable locations at scope boundaries, and any register candidate will be reported as whatever register it happens to occupy at the end of the method.

Reply:
Yes, you are right. I revert that commit. Thanks!